### DISCUSSION TITLE
How can we help you with v.5?

### DISCUSSION THREAD

**Discussion by ascensus-mdb** [twbs/bootstrap#30636] [General] [answered]
Hi Bootstrap Team and again thanks for all the hard, great work :)

With a group of experienced programmers from our company, we were wondering how we could support your effort in developing V5.

We greatly appreciate all your contributions to Bootstrap's development and what it gives to the entire community. We'd love to help you with the development of v.5.

So, is there anything we could do to make your work easier and help to deliver v.5 sooner?

**Comment by mdo** [comment, marked as answer]
So cool to see the excitement and enthusiasm here! Right now we're still cranking away, but we're closing in our first alpha with some major PRs landing to help improve our documentation. You can head to https://twbs-bootstrap.netlify.app to test things out yourself. There are a few known issues there (version and URL being a big one), but testing and feedback is something we'll need the most in the coming months :).
